contracts/src/v0.8/functions/tests/1_0_0/Setup.t.sol

```diff
@@ -265,3 +265,134 @@ contract FunctionsFulfillmentSetup is FunctionsClientRequestSetup {
     vm.startPrank(OWNER_ADDRESS);
   }
 }
+
+contract FunctionsMultipleFulfillmentsSetup is FunctionsFulfillmentSetup {
+  bytes32 s_requestId2;
+  FunctionsResponse.Commitment s_requestCommitment2;
+  bytes32 s_requestId3;
+  FunctionsResponse.Commitment s_requestCommitment3;
+
+  function setUp() public virtual override {
+    FunctionsFulfillmentSetup.setUp();
+
+    // Make 2 additional requests (1 already complete)
+
+    //  *** Request #2 ***
+    vm.recordLogs();
+    s_requestId2 = s_functionsClient.sendRequest(
+      s_donId,
+      "return 'hello world';",
+      new bytes(0),
+      new string[](0),
+      new bytes[](0),
+      s_subscriptionId,
+      5500
+    );
+
+    // Get commitment data from OracleRequest event log
+    Vm.Log[] memory entriesAfterRequest2 = vm.getRecordedLogs();
+    (, , , , , , , FunctionsResponse.Commitment memory commitment2) = abi.decode(
+      entriesAfterRequest2[0].data,
+      (address, uint64, address, bytes, uint16, bytes32, uint64, FunctionsResponse.Commitment)
+    );
+    s_requestCommitment2 = commitment2;
+
+    // Transmit as transmitter 2
+    vm.stopPrank();
+    vm.startPrank(NOP_TRANSMITTER_ADDRESS_2);
+
+    // Build report
+    bytes32[] memory requestIds2 = new bytes32[](1);
+    requestIds2[0] = s_requestId2;
+    bytes[] memory results2 = new bytes[](1);
+    results2[0] = bytes("hello world!");
+    bytes[] memory errors2 = new bytes[](1);
+    // No error
+    bytes[] memory onchainMetadata2 = new bytes[](1);
+    onchainMetadata2[0] = abi.encode(s_requestCommitment2);
+    bytes[] memory offchainMetadata2 = new bytes[](1);
+    // No offchain metadata
+    bytes memory report2 = abi.encode(requestIds2, results2, errors2, onchainMetadata2, offchainMetadata2);
+
+    // Build signers
+    address[31] memory signers2;
+    signers2[0] = NOP_SIGNER_ADDRESS_2;
+
+    // Send report
+    vm.recordLogs();
+    s_functionsCoordinator.callReportWithSigners(report2, signers2);
```

Comment:
How can that work with only one address in signers2 array? Don't we always need 3? I'm confused.

Reply:
The way this is working is bypassing the OCR validation, because the testhelper calls `_report` directly, rather than `transmit`.
I'll go back in and make it run through all that for real next week.